modules/apps/frontend-js/frontend-js-web/build.gradle

```diff
@@ -23,21 +21,8 @@ buildClay {
 	into clayDestinationDir
 }
 
-buildLiferayAMDLoader {
-	dependsOn npmInstall
-	eachFile new StripPathSegmentsAction(4)
-	from npmInstall.nodeModulesDir
-
-	include "@liferay/amd-loader/build/loader/loader.js.map"
-	include "@liferay/amd-loader/build/loader/loader.js"
-
-	includeEmptyDirs = false
-	into new File(jsDestinationDir, "loader")
-}
-
 classes {
 	dependsOn buildClay
-	dependsOn buildLiferayAMDLoader
```

Comment:
This is good. The AMD loader never really belong to this module :+1:

Reply:
We should change [this lines](https://github.com/liferay/liferay-frontend-projects/blob/master/projects/amd-loader/webpack.proxyPortal.js#L13-L14) in the AMD loader project, so that we can keep debugging it.